src/util/use-emily-deposit.ts

```diff
@@ -0,0 +1,38 @@
+import { NotificationStatusType } from "@/comps/Notifications";
+import { useNotifications } from "@/hooks/use-notifications";
+import { useMutation } from "@tanstack/react-query";
+
+const expBackoff = (attempt: number) =>
+  Math.min(attempt > 1 ? 2 ** attempt * 1000 : 1000, 30 * 1000);
```

Comment:
nit: I guess this comes from the doc, but why do we need the ternary? also, nit
```suggestion
  Math.min(2 ** attempt, 30) * 1000;
```